test/conformance/source/environment.cpp

```diff
@@ -199,6 +201,22 @@ DevicesEnvironment::DevicesEnvironment(int argc, char **argv)
         error = "Could not find any devices associated with the platform";
         return;
     }
+    // Get the argument (test_devices_count) to limit test devices count.
+    u_long count_set = 0;
+    for (int i = 1; i < argc; ++i) {
```

Comment:
```suggestion
    for (int i = 1; i < argc - 1; ++i) {
```
and drop the `&& i + 1 < argc` in the if below?

Reply:
You're right. It will work too.